### DISCUSSION TITLE
rank graph too thin

### DISCUSSION THREAD

**Discussion by GitWilf** [ppy/osu#23444] [Ideas]
![image](https://user-images.githubusercontent.com/123104065/236907063-aa35c6d2-c2d9-4970-8cad-27fc0b2bfc4d.png)
too thin i not like it

**Comment by Rekunan** [comment]
I actually prefer it since it clearly shows the steps/days, but it is different from osu!web.

**Reply by bdach** [reply]
The width (in numerical value) is supposed to be the same as web, but due to whatever differences in rendering they end up being not-the-same.

**Reply by GitWilf** [reply]
well i do the OPPOSITE of preferring it! (maybe they could add something in settings, and even then this is so thin its even being entirely alialised)

**Reply by bdach** [reply]
I agree it looks bad but it will never in a million years be a setting.

**Reply by GitWilf** [reply]
i like how you hyphenated "not-the-same"

**Reply by peppy** [reply]
I think the thickness should be increased until the aliasing effects go away.
